recOrder/cli/compute_transfer_function.py

```diff
@@ -34,76 +45,10 @@ def compute_transfer_function_cli(config_path, output_path):
 
     # Pass settings to appropriate calculate_transfer_function and save
     if settings.universal_settings.reconstruct_birefringence:
-        echo_headline(
-            "Generating birefringence transfer function with settings:"
-        )
-        echo_settings(settings.birefringence_transfer_function_settings)
-
-        # Calculate transfer functions
-        intensity_to_stokes_matrix = (
-            inplane_anisotropic_thin_pol3d.calculate_transfer_function(
-                **settings.birefringence_transfer_function_settings.dict()
-            )
-        )
-        # Save
-        dataset[
-            "intensity_to_stokes_matrix"
-        ] = intensity_to_stokes_matrix.cpu().numpy()[None, None, None, ...]
+        dataset = generate_birefringence(settings, dataset)
```

Comment:
I think this refactor is great!

As we discussed, I don't think you need to return `dataset` here. I also think you can rename `generate_birefringence` to `generate_birefringence_transfer_function`.